Source/UI/NewUI/BillingForm/View/BillingFormCellTextField.swift

```diff
@@ -21,8 +21,13 @@ final class BillingFormCellTextField: UITableViewCell {
         return view
     }()
 
+    private lazy var tapGesture: UITapGestureRecognizer = {
+      UIView.keyboardDismissTapGesture
+    }()
```

Comment:
This is used in the initialiser. I've raised this previously and still no outcome, but there is no benefit from making property lazy here, or by complicating the assignment to use a completion handler vs direct value. It does however add lines of code that result in unnecessary noise and distractions...

Reply:
`lazy` because the target is `self` in `UIView` extension

Reply:
How does that work, its a `static` method in the UIView extension, how does `self` exist? ❓